docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx

```diff
@@ -78,6 +78,15 @@ You can reduce the policy size by separating them into multiple IAM roles. Use
   /labels/#auto-discovery)
   for more details.
   </Admonition>
+
+  Create or print the required IAM policies with the following commands and
+  attach them to the respective IAM roles:
+  - `teleport db configure aws create-iam --types redshift,redshift-serverless --name teleport-redshift-access`
+  - `teleport db configure aws print-iam --types redshift,redshift-serverless`
```

Comment:
````suggestion

```code
$ teleport db configure aws create-iam --types redshift,redshift-serverless --name teleport-redshift-access
$ teleport db configure aws print-iam --types redshift,redshift-serverless
```
````
Suggesting making this a `code` block for easier copy/pasting